README.md

```diff
@@ -41,7 +41,22 @@ in your commit messages so it is clear what each commit is for._" <br />
 when you want someone to review your work..._" <br />
 > "_Please only assign a Pull Request for review
 once all the tests have passed on CI..._" <br />
-> For a **list** of these common scenarios see:
+
+We've devised a set of rules and actions to respond to these problems, they look like this:
+
+| Category      | Rule                                                                               | dwylbot comments                                                                                                                                                                                                                                                                                                      | Labels changed                                                                   |
```

Comment:
That's what I had tried to include in the 'rule' column, would you like me to rephrase them? 

I think I called it 'rule' rather than say 'trigger' because in the readme I'm thinking about it from a user over developer perspective... the user wants to know what our gitflow rules are to decide whether they want to use dwylbot because it's useful/relevant to their own gitflow.